superset/utils/manifest_processor.py

```diff
@@ -0,0 +1,69 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import json
+
+
+class UIManifestProcessor:
```

Comment:
Any reason this isn't in the `extenstions.py` file?

Reply:
Would like to keep extensions a bit more light-weight. It will likely end up containing lots of "global" objects as we move along. Also, it might be good to be able to test this independently of instantiating it.

Reply:
From a testing standpoint I’m not certain it matters if this resides in its own file or part in `extensions.py’? Given it’s a Flask extension If I were browsing the code I think I would expect it to be housed with the other extensions.